src/reaper_osara.cpp

```diff
@@ -1010,6 +1010,20 @@ void postToggleCountIn(int command) {
 	outputMessage(GetToggleCommandState(command) ? "count in on":"count in off");
 }
 
+void postChangeTempo(int command) {
+	double tempo = Master_GetTempo();
+	ostringstream s;
+	s << tempo << " BPM";
+	outputMessage(s);
+}
+
+void postTogglePlaybackPositionFollowsTimebase(int command) {
+	ostringstream s;
+	s << "Playback position follows project timebase when changing tempo: " ;
```

Comment:
I worry about efficiency here: this is a very long message to hear before hearing the new state. It's true that OSARA normally says the name before the state; e.g. "repeat on", "metronome on". However, this one really is very long. If this is a command that might be used often, that could be a big efficiency killer.

One way of getting around this could be to say "enabled playback position follows project timebase when changing tempo" (or "disabled" if turning it off). That's a bit inconsistent with some other messages, but there's some prior art; e.g. we say "unmuted" instead of "mute off".

Reply:
I agree, it's done.